TemperatureValidator.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace BatteryManagementSystem
+{
+    public class TemperatureValidator : IValidator<BatteryMonitor>
```

Comment:
Check validators ....duplicate code

Reply:
also this i need bit more explanation what can be done better.